client-js/app/scripts/activities.ts

```diff
@@ -370,17 +370,16 @@ export class ManagerActivity implements Activity {
               private api: manager.ManagerApi) {}
 
   async start(): Promise<ActivityState> {
-    var conns = await this.api.connections();
-    var connsWithUrl = conns.map((c) => {
-      var x = <any> c;
+    var connections = await this.api.connections();
```

Comment:
Would it be nicer to deduplicate this code by just having `updateConnections` called on attach?